content/blog/cozycoder-vision/index.md

```diff
@@ -0,0 +1,69 @@
+
+Of course, not every company creates so many new projects. But this is just one example to show what could be possible, if we were using LLMs more efficiently.
+
+## Designed for your Laptop not the Cloud
+
+The above example gives a rough idea of what you as a software developer should be able to do with Cozy Coder. But to make the tool really good, it obviously must run on your laptop.
+
+Many developers use CLIs and they can easily be used within scripts to automate tasks. As Cozy Coder is designed for developers, it must provide a proper CLI. But also the access to the file system is important to directly modify local files, if you want to.
+
+Even more important are the constraints of large enterprises. In the last decade, I haven't worked for a single large company here in Europe that did not host their own Github Enterprise or Gitlab or similar tool. The code is capitalized and must to be kept secret. Hosting the code on a self-hosted Github Enterprise that is only accessible from your VPN is how you achieve this. So why should one of those companies start sending code to some LLM hosted in the cloud? Everything that runs locally on your laptop is significantly easier to use within larger enterprises.
+
+Cozy Coder will of course not be limited to local LLMs, but they will always be the preferred choice and well supported. Ideally we will all together achieve the usage of local LLMs, where possible and only use the cloud based ones, where necessary.
+
+Of course, Cozy Coder will also have a server side part. But it will be mainly for enabling sharing and collaboration as well backups of data.
+
+## Commercial Open Source
+
+Cozy Coder is an open source project and I wouldn't develop it, if it wasn't. But with the end of zero interest rate policy, sponsoring as a strategy to fund an open source project has become more difficult.
+
+That's why the goal is to have a more sustainable financing strategy by building a for-profit business on top of the open source project. This will be achieved by following the open core approach. While Cozy Coder itself will be available under Apache-2.0 license and can be used for free, the server side code will be licensed under a Polyform non-commercial license. While it still allows personal or educational use and even code modification, the commercial use requires a paid license.
```

Comment:
```suggestion
That's why the goal is to have a more sustainable financing strategy by building a for-profit business on top of the open source project. This will be achieved by following the open core approach. While Cozy Coder itself will be available under Apache-2.0 license and can be used for free, the server side code will be licensed under a Polyform non-commercial license. Thus, it still allows personal or educational use and even code modification, the commercial use requires a paid license.
```

Double sentences using `while` transition word. I replaced it with one that seemed to fit. Feel free to swap it out :)